microcosm_postgres/store.py

```diff
@@ -217,6 +217,7 @@ def _delete(self, *criterion):
         """
         with self.flushing():
             count = self._query(*criterion).delete()
+            self.session.commit()
```

Comment:
We probably don't want the persistence layer to make decisions about transactions.